tests/UnishTestCase.php

```diff
@@ -107,6 +107,9 @@ public static function cleanDirs()
                     self::recursiveDelete($target);
                 }
             }
+            // Bring back any committed assets removed above.
+            $webroot = self::webroot();
+            exec("git -C $webroot checkout -- .");
```

Comment:
Clever! But can we guarantee that we have a git clone? Can't someone start with a tarball from composer? Maybe thats something we don't care about?

Reply:
Hm good point. The code as written deletes the Drush commandfiles I added. I was also vaguely concerned that there might be other necessary items that were getting deleted. It might be better to just be careful about what we delete in the cleanup step.